dev/create_release_tag.py

```diff
@@ -26,6 +26,7 @@
 git checkout master
 git branch -D branch-9.0
 """
+
```

Comment:
There is no flag to disable inserting a new line after module docstring.

Reply:
Thanks for the review! Could you give me a hint what should I do with this line?

Reply:
Sorry, never mind. Just a note for myself.